x/bank/types/balance.go

```diff
@@ -55,7 +55,7 @@ func (b Balance) Validate() error {
 	}
 
 	// sort the coins post validation
-	b.Coins = b.Coins.Sort()
+	b.Coins = b.Coins.Sort() // nolint
```

Comment:
I dont think this does anything here.  Since the method doesnt use a pointer nor return the value. cc @alexanderbez for sanity check